src/gui/accountstate.h

```diff
@@ -223,6 +238,13 @@ protected Q_SLOTS:
      */
     AccountAppList _apps;
 
+    /**
+     * Websocket connection for the account
+     */
+    QWebSocket webSocket;
```

Comment:
Yes, by pointer please. Also the coding style in the project is to prefix member variables with `_`.